daemon/rpc_set_lan_discovery_test.go

```diff
@@ -224,74 +215,3 @@ func TestSetLANDiscovery_Error(t *testing.T) {
 		})
 	}
 }
-
-func TestSetLANDiscovery_MeshInteraction(t *testing.T) {
```

Comment:
Why this test is removed?

Reply:
Because this was basically testing whether the correct mesh peers are being passed to networker and after my changes mesh peers are not being passed at all anymore.